src/data/data.js

```diff
@@ -0,0 +1,93 @@
+export const Home = {
+    heroTitle: "Welcometo The Scrum Guide",
```

Comment:
"Welcome to" is not correctly spelled <3

Reply:
fixed mistake